LMP 3.5 install-feature failed if using OL nightly build

To reproduce the problem:
- `git clone https://github.com/openliberty/guide-cdi-intro.git`
- `cd guide-cdi-intro/finish`
- update pom.xml by adding a nightly build
```
             <plugin>
                 <groupId>io.openliberty.tools</groupId>
                 <artifactId>liberty-maven-plugin</artifactId>
<configuration><install><runtimeUrl>https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/runtime/nightly/2021-11-23_1100/openliberty-all-21.0.0.12-cl220120211123-1100.zip</runtimeUrl></install></configuration>
                 <version>3.5</version>
             </plugin>
```
- run `mvn package liberty:create liberty:install-feature` and got following error:
```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  12.731 s
[INFO] Finished at: 2021-11-24T15:59:57-05:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal io.openliberty.tools:liberty-maven-plugin:3.5:install-feature (default-cli) on project guide-cdi-intro: Could not install feature -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```


Note:
- `mvn clean liberty:run` has this same problem
- LMP 3.4 has no problem

Expectation:
- no error and no need to install features because the build `openliberty-all-21.0.0.12-cl220120211123-1100.zip` contains all features.

--- a/liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/InstallFeatureMojo.java
+++ b/liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/InstallFeatureMojo.java
@@ -85,10 +85,7 @@ public class InstallFeatureMojo extends InstallFeatureSupport {
         }
         else if(util != null) {
             util.installFeatures(features.isAcceptLicense(), new ArrayList<String>(featuresToInstall));
-        }else if(util == null) {
-        	throw new IllegalStateException(MessageFormat.format("Could not install feature", "install-feature"));
         }
-        
        
     }
 

